live-examples/html-examples/article.html

```diff
@@ -0,0 +1,15 @@
+<article class="forecast">
+    <h1>Weather forecast for Seattle</h1>
+    <article class="day-forecast">
+        <h2>03 March 2018</h2>
+        <p>Rain.</p>
+    </article>
```

Comment:
This makes sense to me. Each of these can stand on their own. It does loose some context without the heading though.